builtin/logical/mssql/path_config_lease.go

```diff
@@ -78,7 +87,7 @@ func (b *backend) pathConfigLeaseRead(
 	return &logical.Response{
 		Data: map[string]interface{}{
 			"ttl":     leaseConfig.TTL.String(),
-			"ttl_max": leaseConfig.TTLMax.String(),
+			"max_ttl": leaseConfig.TTLMax.String(),
```

Comment:
This is the only thing I'm potentially nervous about. I don't expect this change to be a significant burden on anyone but sometimes you never know. The alternative though is to add a bunch more code to figure out which parameter they used initially.

@vishalnayak thoughts?